save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/ProjectView.kt

```diff
@@ -699,6 +712,44 @@ class ProjectView : AbstractView<ProjectExecutionRouteProps, ProjectViewState>(f
         }
     }
 
+    private fun updateGit() {
```

Comment:
Why do we need all this logic here and not inside settings view? It'll make code more isolated if you pass only the callback to update state of parent component instead of placing the entire logic of the request

Reply:
moved